doc/api/repl.md

```diff
@@ -321,17 +326,17 @@ The following example shows two new commands added to the REPL instance:
 ```js
 const repl = require('repl');
 
-var replServer = repl.start({prompt: '> '});
+const replServer = repl.start({prompt: '> '});
 replServer.defineCommand('sayhello', {
   help: 'Say hello',
-  action: function(name) {
+  action: function sayHello(name) {
```

Comment:
doesn't add any value. I'd rather just leave at is it, but do `action: function (name) {` for coding style's sake